docs/appkit/react/wagmi/hooks.mdx

```diff
@@ -2,22 +2,6 @@ import Button from '../../../components/button'
 
 You can use [Wagmi hooks](https://wagmi.sh/react/hooks/useAccount) to sign messages, interact with smart contracts, and much more.
 
-### useAccount
-
-Hook for accessing account data and connection status.
-
-```tsx
-import { useAccount } from 'wagmi'
-
-function App() {
-  const { address, isConnecting, isDisconnected } = useAccount()
-
-  if (isConnecting) return <div>Connecting…</div>
-  if (isDisconnected) return <div>Disconnected</div>
-  return <div>{address}</div>
-}
-```
-
```

Comment:
Can't we still use this Wagmi function? 

This should still work (I tested it and can confirm that it works), can you please confirm? I don't think we should remove this from the docs.